app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}"
+                )
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False)\
```

Comment:
it's better to move parameters on the next line(including the first one). And also we don't use \